src/renderer/services/dex.js

```diff
@@ -1465,6 +1468,251 @@ export default {
+          });
+      } catch (e) {
+        reject(`Failed to fetch commit state. ${e.message}`);
+      }
+    });
+  },
+
+  fetchCommitDEXState() {
+    return new Promise((resolve, reject) => {
+      try {
+        const rpcClient = network.getRpcClient();
+        rpcClient.query({
+          method: 'getstorage',
+          params: [assets.DEX_SCRIPT_HASH, `${u.reverseHex(assets.APH)}fa`],
+        })
+          .then((res) => {
+            const dexState = {
+              totalUnitsContributed: 0,
+              lastAppliedFeeSnapshot: 0,
+              totalFeeUnits: 0,
+            };
+
+            if (res.result && res.result.length >= 48) {
+              dexState.totalUnitsContributed = u.fixed82num(res.result.substr(0, 16)) * 100000000;
+              dexState.lastAppliedFeeSnapshot = u.fixed82num(res.result.substr(16, 16));
+              dexState.totalFeeUnits = u.fixed82num(res.result.substr(32, 16));
+            }
+
+            rpcClient.query({
+              method: 'getstorage',
+              params: [assets.DEX_SCRIPT_HASH, `${u.reverseHex(assets.APH)}fc`],
+            })
+              .then((res) => {
+                if (!res || !res.result || res.result.length < 32) {
+                  dexState.totalFeesCollected = 0;
+                  resolve(dexState);
+                  return;
+                }
+
+                dexState.totalFeesCollected = u.fixed82num(res.result.substr(0, 16));
+                // console.log(`dexState.totalFeesCollected: ${dexState.totalFeesCollected}`);
+                resolve(dexState);
+              })
+              .catch((e) => {
+                reject(`Failed to fetch commit state. ${e.message}`);
+              });
+          })
+          .catch((e) => {
+            reject(`Failed to fetch commit state. ${e.message}`);
+          });
+      } catch (e) {
+        reject(`Failed to fetch commit state. ${e.message}`);
+      }
+    });
+  },
+
+  commitAPH(quantity) {
+    return new Promise((resolve, reject) => {
+      try {
+        this.executeContractTransaction('commit',
+          [
+            u.num2fixed8(quantity),
+          ])
+          .then((res) => {
+            if (res.success) {
+              alerts.success('Commit relayed, waiting for confirmation...');
+              neo.monitorTransactionConfirmation(res.tx, true)
+                .then(() => {
+                  const inMemoryHolding = _.get(store.state.nep5Balances, assets.APH);
+                  if (inMemoryHolding) {
+                    inMemoryHolding.balance = null;
+                  }
+
+                  resolve(res.tx);
+                })
+                .catch((e) => {
+                  reject(e);
+                });
+            } else {
+              reject('Transaction rejected');
+            }
+          })
+          .catch((e) => {
+            reject(`Commit Failed. ${e.message}`);
+          });
+      } catch (e) {
+        reject(`Commit Failed. ${e.message}`);
+      }
```

Comment:
this.$t()

Reply:
There are a lot of missing translations in the dex.js / neo.js for error messages. Don't think those are going to make it in this PR due to time constraints. We can make a ticket for it and do them separately.